scripts/dev_cli.py

```diff
@@ -218,11 +248,6 @@ def run(self, args: argparse.Namespace):
                     + ["--db", "ims", "--archive", "--drop"],
                     stdin=file,
                 )
-        else:
```

Comment:
Please remove

```
        parser.add_argument(
            "-g",
            "--generated",
            action="store_true",
            help="Specify this flag to import all generated test data instead of just standard data e.g. units",
        )
```
and
```
if args.generated:
```
so this is the default.